src/formElementPlugins/SummaryViewWrapper.tsx

```diff
@@ -1,12 +1,58 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import { ErrorBoundary, pluginProvider } from '.'
+import { Grid, Icon } from 'semantic-ui-react'
+import { SummaryViewWrapperProps, PluginComponents, ValidationState } from './types'
 import { Form, Header } from 'semantic-ui-react'
-import { SummaryViewWrapperProps, PluginComponents } from './types'
 import { TemplateElementCategory } from '../utils/generated/graphql'
+import { defaultValidate } from './defaultValidate'
+import { EvaluatorParameters } from '../utils/types'
+import { IQueryNode } from '@openmsupply/expression-evaluator/lib/types'
 
 const SummaryViewWrapper: React.FC<SummaryViewWrapperProps> = (props) => {
-  const { element, response } = props
+  const { element, response, allResponses, isStrictValidation } = props
   const { parameters, category, code, pluginCode, isEditable, isRequired, isVisible } = element
+  const { validation: validationExpression, validationMessage } = parameters
+  const [validationState, setValidationState] = useState<ValidationState>({} as ValidationState)
+  const responses = { thisResponse: response?.text, ...allResponses }
+  const [pluginMethods, setPluginMethods] = useState({
+    validate: (
+      validationExpress: IQueryNode,
+      validationMessage: string,
+      evaluatorParameters: EvaluatorParameters
+    ): any =>
+      defaultValidate(validationExpression, validationMessage, {
+        objects: [responses],
+        APIfetch: fetch,
+      }),
+  })
+
+  useEffect(() => {
+    if (!pluginCode) return
+    // TODO use plugin-specific validation method if defined
+    setPluginMethods({
+      validate: defaultValidate,
+    })
+  }, [])
```

Comment:
Same as `ApplicationViewWrapper` maybe add a comment to explain what this effect is for

Reply:
See above.

Reply:
Yes. Can the comment be added here as well please?